cmd/gci/README.md

```diff
@@ -0,0 +1,85 @@
+# gci
+
+`gci`, a tool that control golang package import order and make it always deterministic.
+
+It is from [repo](https://github.com/daixiang0/gci) v0.1 release version.
+
+## Examples
+
+Run `gci -w -local github.com/daixiang0/gci main.go` and you will handle following cases.
+
+### simple case
+
+```go
+package main
+import (
+  "golang.org/x/tools"
+
+  "fmt"
+
+  "github.com/daixiang0/gci"
+)
+```
+
+to
+
+```go
+package main
+import (
+  "fmt"
+
+  "golang.org/x/tools"
+
+  "github.com/daixiang0/gci"
+)
+```
+
+### with alias
+
+```go
+package main
+import (
+  "fmt"
+  go "github.com/golang"
+  "github.com/daixiang0"
+)
+```
+
+to
+
+```go
+package main
+import (
+  "fmt"
+
+  go "github.com/golang"
+
+  "github.com/daixiang0/gci"
+)
+```
+
+### with comment and alias
+
+```go
+package main
+import (
+  "fmt"
+  _ "github.com/golang" // golang
```

Comment:
could you add both `inline` comment and `above line` comment examples?

Reply:
I have added `above line` comment, could you share `inline` comment, i am confused about it.

Reply:
i think you already have it: `_ "github.com/golang" // golang` here

Reply:
Now all done :)